src/runners/handlers/jira.py

```diff
@@ -144,7 +149,7 @@ def bail_out(alert_id):
         log.error(e, f"Failed to update alert {alert_id} with handler status")
 
 
-def handle(alert, correlation_id, project=PROJECT):
+def handle(alert, correlation_id, project=PROJECT, assignee=None, custom_field=None):
```

Comment:
There's probably a nice way to use kwargs here?

Reply:
You could, but it might break things unexpectedly (e.g. if extra params at in the alert definition, it's better to ignore them rather than trying to pass them into `create_jira_ticket` and throwing an exception).